dsacalib/ms_io.py

```diff
@@ -1006,8 +1013,10 @@ def caltable_to_etcd(msname, calname, caltime, status, pols=None, logger=None):
         de.put_dict(f"/mon/cal/{antnum + 1}", dd)
 
 
-def get_antenna_gains(gains, ant1, ant2, refant=0):
-    """Calculates antenna gains, g_i, from CASA table of G_ij=g_i g_j*.
+def get_antenna_gains(
```

Comment:
In the docstring, describe the relation between casa gains and the antenna gains.